src/frontend/org/voltdb/utils/CSVLoader.java

```
@@ -155,18 +154,16 @@ public void run() {
                     }
                     out_invaliderowfile.write(currItem.errorInfo[0] + "\n");
                     String message = "Invalid input on line " + currItem.lineNumber + ". " + currItem.errorInfo[1];
-                    m_log.error(message);
                     out_logfile.write(message + "\n  Content: " + currItem.errorInfo[0] + "\n");
-
                     m_errorCount++;
 
-                } catch (FileNotFoundException e) {
-                    m_log.error("CSV report directory '" + config.reportdir
-                            + "' does not exist.");
                 } catch (Exception x) {
-                    m_log.error(x.getMessage());
+                    try {
+                        out_logfile.write(x.getMessage());
+                    } catch (IOException e) {
+                        e.printStackTrace();
```

We want to print the original message to std error, not the exception that happened trying to write it to log file.

---

Actually, thinking about this again, previously this error used to go to volt.log and console. So you can ignore this exception (do nothing if this IOexception happens) and add a System.stderr.println  for x.getMessage.